mni_atlas_reader/atlas_reader.py

```diff
@@ -557,5 +566,5 @@ def main():
                   probabilityThreshold=opts.probabilityThreshold)
 
 
-if __name__ == "__main__":
```

Comment:
Thanks for catching that. I don't like `"`. Uses twice as much space as `'`...

Reply:
Yes! Consistency is 🔑